lib/src/mock_confirmation_result.dart

```diff
@@ -2,12 +2,12 @@ import 'package:firebase_auth/firebase_auth.dart';
 import 'package:mockito/mockito.dart';
 
 class MockConfirmationResult extends Mock implements ConfirmationResult {
-  Function onConfirm;
+  Function? onConfirm;
 
   MockConfirmationResult({this.onConfirm});
```

Comment:
`onConfirm` should be non-nullable, and the argument should be `required` in the constructor. Otherwise, the way you wrote it on line 11, it is actually possible to throw a null pointer exception. See https://dartpad.dev/52b00be7de331d37adce5c4d210c5c48?null_safety=true.

![image](https://user-images.githubusercontent.com/3399854/111017033-7568e680-83f4-11eb-942a-33f419a4ecff.png)

Reply:
Since this is very minor, I've merged your PR, and will do the change myself.